com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/FunctionDescriptor.cs

```diff
@@ -28,5 +28,16 @@ namespace com.unity.shadergraph.defs {
             Body = body;
         }
 
+        public FunctionDescriptor(
+        int version,
+        string name,
+        string body,
+        params ParameterDescriptor[] parameters)
+        {
+            Version = version;
+            Name = name;
+            Parameters = parameters.ToList().AsReadOnly();
+            Body = body;
+        }
```

Comment:
Added an alternate constructor that just use the params variadic instead of an IEnumerable.